config/environment.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  return {
+    emberResponsiveDefaults: {
+      autoInject: true
```

Comment:
I originally made this using a list of factories, but I think the end user can inject with their own initializers eventually. What factory was it that caused the bug you had?